lib/compat/wordpress-6.8/class-gutenberg-rest-static-templates-controller.php

```diff
@@ -118,11 +118,33 @@ public function register_routes() {
 	}
 
 	public function get_items( $request ) {
+		$query = array();
+		if ( isset( $request['area'] ) ) {
```

Comment:
Is this one needed? I thought it was just a template part thing?

Reply:
I have no idea what this is, I just copied it from

https://github.com/WordPress/wordpress-develop/blob/ed0ad98bc408a4abaeda00faf00be87159635d1e/src/wp-includes/rest-api/endpoints/class-wp-rest-templates-controller.php#L272-L281

I can remove it if it's not relevant.